samples/vpc_sc_project_perimeter_v1_blacklist.yaml

```diff
@@ -16,6 +16,8 @@ apiVersion: constraints.gatekeeper.sh/v1alpha1
 kind: GCPVPCSCProjectPerimeterConstraintV1
```

Comment:
We could probably just remove this sample entirely?

Reply:
Removed.